spec/controllers/responsible_body/users_controller_spec.rb

```diff
@@ -47,4 +47,33 @@
       end
     end
   end
+
+  describe '#create' do
+    before do
+      FeatureFlag.activate(:rbs_can_manage_users)
```

Comment:
I don't think you should need this flag for creating users?

Reply:
good spot, i've now removed this line